theme/less/_homepage.less

```diff
@@ -53,7 +66,25 @@
   box-shadow: inset 0 0 8px 4px #d4d4d4;
 }
 
-.box a:visited
+.box a:hover, .box a:active, .box a:visited
 {
   color: inherit;
 }
+
+@media screen and (max-width: 770px) 
```

Comment:
I added media query because `grid-column`  doesn't provide automatic features for column number rearranging like it was in case of `repeat()`.